kore/src/Kore/Variables/Free.hs

```diff
@@ -26,6 +27,9 @@ import qualified Data.Set as Set
 
 import Kore.Syntax
 
+unlessM :: Monad m => m Bool -> m () -> m ()
```

Comment:
The "standard" place to import this is from [`Extra`](https://www.stackage.org/haddock/lts-13.28/extra-1.6.17/Extra.html#v:unlessM).